Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -43,11 +55,11 @@ struct CreateBuffersAroundPointsView: View {
             .toolbar {
                 ToolbarItemGroup(placement: .bottomBar) {
                     // Union toggle switch.
-                    Toggle("Union", isOn: $model.shouldUnion)
+                    Toggle("Union", isOn: $shouldUnion)
                         .toggleStyle(.switch)
-                        .onChange(of: model.shouldUnion) { _ in
+                        .onChange(of: shouldUnion) { _ in
                             if !model.bufferPoints.isEmpty {
-                                model.drawBuffers()
+                                model.drawBuffers(shouldUnion: shouldUnion)
                             }
                         }
```

Comment:
The action closure should use the value passed to it. Also, not a big deal, but a guard statement could be used.

```swift
.onChange(of: shouldUnion) { newShouldUnion in
    guard !model.bufferPoints.isEmpty else { return }
    model.drawBuffers(shouldUnion: newShouldUnion)
}
```